script/Arcueid/Day 7/07_00_ARC07_3.txt

```diff
@@ -151,38 +160,46 @@ Reflexively and automatically, I run toward the demonic living room as fast as I
 [sha:7a84eaedbb1dbb1f12b789b51e6ea483263d4942]{
 -- Page 178, Offset 17016.
 -- 力いっぱい扉を開けて中に入る。
-I slam the door open with all my strength and enter the room.
+I throw the door wide and burst into the room.
 }
 [sha:5d59196c3941c46566a4da141e6e077c041deb58]{
 -- Page 179, Offset 17017.
 -- だが、もう何もかも遅かった。
-But it was too late.
+Alas, it's already too late.
 }
 [sha:4b77f98899478523839265c725944221fcf21ed7]{
 -- Page 179, Offset 17018.
 -- 事態は収拾不可能な、驚天動地……俺のライフパスにおいて……の展開を迎えていた。
-// 事態は収拾不可能な、驚天動地……俺のライフパスにおいて……の展開を迎えていた。
Recheck please
-I come face-to-face with the uncontrollable, earth-shattering latest development to the story of my life.
+// Alright, so seems like ライフパス is... a Cyberpunk TTRPG reference?
```

Comment:
weblio supports this being a TTRPG thing.
> ライフパス(lifepath) は、テーブルトークRPGで、プレイヤーキャラクターのそれまでの人生の経歴をあらわすデータである

I might adjust the TL a bit though so it's "I'm greated by the latest twist in my character development" or "character background"